### ISSUE TITLE
Mood Congruent/Mood Incongruent Modifiers

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#10171]
We need two modifiers for mood congruence and mood incongruence.

**Preferred term label:**
Mood congruence
Mood incongruence

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by bcoleman96** [comment]
I need to continue to edit this

**Comment by pnrobinson** [comment]
@bcoleman96  -- please provide more information!

**Comment by pnrobinson** [comment]
@bcoleman96 any updates?
Is it true that this only applies to Delusion and Hallucination?
This article has the definitions from DSM
https://www.verywellmind.com/mood-incongruent-380034
I am not sure that congruence/incongruence even applies to all forms of [Delusion](https://hpo.jax.org/app/browse/term/HP:0000746)? Is it perhaps better to have just a regular HPO term called Incongruent delusion?

**Comment by pnrobinson** [comment]
@bcoleman96 
Let's decide this at the Budapest workshop

**Comment by pnrobinson** [comment]
I think the decision is not to add modifiers at this time, as it would be the only two behavior terms with modifiers.
